### DISCUSSION TITLE
Add Default value in Filters

### DISCUSSION THREAD

**Discussion by Naagarjunaa** [Power-Components/livewire-powergrid#704] [Q&A]
Hi team,
How can i add a default value for the filter...
for example, i need to show all data but i need to show only Active status on page load if they need Inactive status they can remove the filter... can you please help me to do that.

![image](https://user-images.githubusercontent.com/46925151/199957678-bc100568-f5c0-4744-b07f-ba21cf1dc0d4.png)

Thanks in Advance.

**Comment by luanfreitasdev** [comment]
Hello. Wouldn't it be makeSelectFilter in this case?

**Reply by Epitaffio** [reply]
We also would need this.

The user load the table for the first time and some filters are set automatically, then they can remove or change them. Like an optional parameter in the makeInput* methods maybe?
